website/docs/en/plugins/rspack/swc-js-minimizer-rspack-plugin.mdx

```diff
@@ -49,6 +49,7 @@ new rspack.SwcJsMinimizerRspackPlugin(options);
     extractComments?: boolean | RegExp;
     compress?: TerserCompressOptions | boolean;
     mangle?: TerserMangleOptions | boolean;
+    module?: boolean;
```

Comment:
Would you please also copy this to the Chinese version?

Reply:
This should also be added to the ts typing.

Reply:
This option is already in the TS declaration.
<img width="383" alt="image" src="https://github.com/web-infra-dev/rspack/assets/31957758/0dc06606-ede7-4619-873e-3159f59ce2cb">